src/NServiceBus.Core.Tests/Sagas/InvokeSagaNotFoundBehaviorTests.cs

```diff
@@ -44,6 +45,42 @@ static Task SetSagaNotFound(IIncomingLogicalMessageContext context)
             return TaskEx.CompletedTask;
         }
 
+        public class DummySagaToProvideMetadata : Saga<DummySagaToProvideMetadata.SagaData>, IHandleMessages<StartSaga>, IAmStartedByMessages<MessageToSaga>
```

Comment:
This class wasn't necessary in https://github.com/Particular/NServiceBus/pull/6248 for 8.x – Is it needed in the 7.7 backport?

Reply:
No, it's not needed.  We missed one of the commits when cherry-picking.  I'll fix this.